plugins/modules/helm.py

```diff
@@ -390,6 +400,7 @@ def main():
             release_namespace=dict(type='str', required=True, aliases=['namespace']),
             release_state=dict(default='present', choices=['present', 'absent'], aliases=['state']),
             release_values=dict(type='dict', default={}, aliases=['values']),
+            values_files=dict(type='list'),
```

Comment:
```suggestion
            values_files=dict(type='list', default=[], elements='str'),
```

`elements='str'` make sure that we have only string values for `values_files`.

Reply:
Also, this will solve CI failures - 

```
ERROR: Found 2 validate-modules issue(s) which need to be resolved:
ERROR: plugins/modules/helm.py:0:0: doc-elements-mismatch: Argument 'values_files' in argument_spec does not specify elements,but elements is documented as being str
ERROR: plugins/modules/helm.py:0:0: parameter-list-no-elements: Argument 'values_files' in argument_spec defines type as list but elements is not defined
```

Reply:
fixed